cars/app/controllers/brands_controller.rb

```diff
@@ -0,0 +1,38 @@
+class BrandsController < ApplicationController
+    def index
+        @brands = Brand.all
+    end
+    
+    def new
+    end
+    
+    def create
+        brand = Brand.new
+        brand.name = params[:name]
+        brand.country = params[:country]
+        brand.save
+        redirect_to '/brands'
+    end
+    
+    def edit
+        @edit = Brand.find_by(id: params[:id])
+    end
+    
+    def update
+        brand = Brand.find_by(id: params[:id])
+        brand.name = params[:name]
+        brand.country = params[:country]
+        brand.save
+        redirect_to '/brands'
+    end
+    
+    def delete
+        brand = Brand.find_by(id: params[:id])
+        brand.delete
+        redirect_to '/brands'
+    end
+    
+    def show
+        @brand = Brand.find_by(id: params[:id])
+    end
+end
```

Comment:
If you want to refactor this a bit you can always add a before filter to all your actions that are finding  a brand by id `Brand.find_by(id: params[:id])`. [Here](https://stackoverflow.com/questions/45615591/how-does-only-at-before-action-work-in-rails) is a good explanation on stackoverflow on how it works.